src/ledger/NetworkConfig.h

```diff
@@ -23,15 +23,15 @@ class Application;
 struct MinimumSorobanNetworkConfig
 {
     static constexpr uint32_t TX_MAX_READ_LEDGER_ENTRIES = 3;
-    static constexpr uint32_t TX_MAX_READ_BYTES = 5'000;
+    static constexpr uint32_t TX_MAX_READ_BYTES = 3'000;
 
     static constexpr uint32_t TX_MAX_WRITE_LEDGER_ENTRIES = 2;
-    static constexpr uint32_t TX_MAX_WRITE_BYTES = 2'000;
+    static constexpr uint32_t TX_MAX_WRITE_BYTES = 3'000;
 
-    static constexpr uint32_t TX_MAX_SIZE_BYTES = 10'000;
+    static constexpr uint32_t TX_MAX_SIZE_BYTES = 5'000;
 
-    static constexpr uint32_t TX_MAX_INSTRUCTIONS = 5'000'000;
-    static constexpr uint32_t MEMORY_LIMIT = 5'000'000;
+    static constexpr uint32_t TX_MAX_INSTRUCTIONS = 2'000'000;
```

Comment:
@dmkozh The upgrade test needs around 1476437. I'm wondering if there's a risk with not including enough of a cushion in case we update some of the metering settings that end up increases the total instruction count (this is possible right?).

Reply:
Set to 2.5M